src/libs/AxelarQueryAPI.ts

```diff
@@ -119,10 +120,12 @@ export class AxelarQueryAPI {
    * @returns base fee in native token in wei, translated into the native gas token of choice
    */
   public async getNativeGasBaseFee(
-    sourceChainName: EvmChain,
-    destinationChainName: EvmChain,
+    sourceChainName: EvmChain | string,
```

Comment:
Maybe it's best to use `string` instead of `EvmChain | string` because EvmChain can be outdated.

Reply:
I think having `EvmChain` here is still useful for auto-completion for the dev.